tests/input/test_input_gg_checks.py

```diff
@@ -17,29 +17,6 @@ def test_parse_config_gg_check_in_file_context_contains_file():
     assert "file.py" in output[0].gg_args
 
 
-def test_parse_config_check_gg_matchfilefragment():
```

Comment:
Why was this test deleted? Is there a good replacement?

Reply:
@PCain02 Thanks for the feedback! This test has been added back in and coverage is maintained.